File: task2.js
//#region  Task1
// let num = 256;
// let i = 2;

// while (num >= i) {
//   i = i * 2;
//   if (num== i) {
//     console.log("2 ededin quvvetidir");
//     break;
//   } else if (a < i) {
//     console.log("2 ededin quvveti deyil");
//   }
// }
//#endregion

//#region task3

const num = 15;
let i = 1;
let count = 0;

while (i <= num) {
  if (num % i == 0) {
    count++;
  }
  i++;
}
if (count == 2) {
  console.log("sade ededir");
} else {
  console.log("murekkeb ededir");
}
//#endregion

//#region Task4

// const month = 4;

// switch (month) {
//   case 1:
//   case 2:
//   case 3:
//     console.log("Winter");
//     break;
//   case 4:
//   case 5:
//   case 6:
//     console.log("Spring");
//     break;
//   case 7:
//   case 8:
//   case 9:
//     console.log("Summer");
//     break;
//   case 10:
//   case 11:
//   case 12:
//     console.log("Autumn");
//     break;
//   default:
//     console.log("Error");
// }

//#endregion

File: task2.cs
#region  Task1
//int num = 256;
//int i = 2;

//while (num >= i)
//{

//    if (num == i)
//    {
//        system.console.writeline("2 ededin quvvetidir"); 

//    }
//    else if (num < i)
//    {
//        system.console.writeline("2 ededin quvveti deyil"); 
//    }
//    i = i * 2;
//}
#endregion
#region Task 2
int number = 26;
int tNumber = 0;
int rem = 0;
int rev = 0;
 tNumber=number


while (number > 0)
{
    rem = number % 10;
    rev = rev * 10 + rem;
    number = number / 10;

}


if (rev == tNumber)
    Console.WriteLine("Given Number is Palindrome");
else
    Console.WriteLine("Given Number is not a Palindrome");
#endregion
#region task3

//int num = 15;
//int i = 1;
//int count = 0;

//while (i <= num)
//{
//    if (num % i == 0)
//    {
//        count++;
//    }
//    i++;
//}
//if (count == 2)
//{
//    System.Console.WriteLine("sade ededir"); 
//}
//else
//{
//    System.Console.WriteLine("murekkeb ededir");
//}
#endregion
#region Task4

// int month = 4;

//switch (month)
//{
//    case 1:
//    case 2:
//    case 3:
//        System.Console.WriteLine("Winter");
//        break;
//    case 4:
//    case 5:
//    case 6:
//        System.Console.WriteLine("Spring");
//        break;
//    case 7:
//    case 8:
//    case 9:
//        System.Console.WriteLine("Summer");
//        break;
//    case 10:
//    case 11:
//    case 12:
//        System.Console.WriteLine("Autumn");
//        break;
//    default:
//        System.Console.WriteLine("Error");
//}

#endregion



